tutorials/scripting/resources.rst

```diff
@@ -238,6 +240,8 @@ Let's see some examples.
                 [Export]
                 public String[] Strings { get; set; }
 
+                // Make sure that every variable gets a default value. 
+                // Otherwise, there will be problems with creating and editing your resource via the inspector.
```

Comment:
```suggestion
                // Make sure that every parameter has a default value. 
                // Otherwise, there will be problems with creating and editing
                // your resource via the inspector.
```